### ISSUE TITLE
Unable to create child window for native control host. Application manifest with supported OS list might be required.

### ISSUE THREAD

**Issue by derekantrican** [AvaloniaDeveloperOrg/Avalonia.WebView#19]
Getting this when trying to show the control:

> System.InvalidOperationException: 'Unable to create child window for native control host. Application manifest with supported OS list might be required.'

This is being thrown in Program.cs at `BuildAvaloniaApp().StartWithClassicDesktopLifetime(args);`

I am using version 11.0.0.1 of WebView.Avalonia & WebView.Avalonia.Desktop. Version 11.0.0 of Avalonia UI.

**Comment by derekantrican** [comment]
Found out I needed app.manifest. Might be helpful to add that to the README

**Comment by ChisterWu** [comment]
Sorry for replying to you so late, Now have you solved this problem?

**Comment by derekantrican** [comment]
Yes, I solved this. I needed to add an app.manifest file to my project
